_data/blogs.yml

```diff
@@ -51,3 +51,9 @@
   blog_img:
   article: GCI 2016
   article_link: http://dcongtinh.blogspot.com/2016/12/google-code-in-2016-google-code-in.html
+  
+- name: Manraj Juneja
+  blog: https://www.blogger.com/blogger.g?blogID=3436867633936758241#editor/target=post;postID=1961075658112278688;onPublishedMenu=allposts;onClosedMenu=allposts;postNum=0;src=postname
+  blog_img:
+  article: GCI 2016
+  article_link:https://wordpress.com/post/blogging157.wordpress.com/17
```

Comment:
space missing after `article_link:`.

Change `line 59` to `article_link: https://wordpress.com/post/blogging157.wordpress.com/17`